src/main/scala/mesosphere/marathon/api/akkahttp/v2/InfoController.scala

```diff
@@ -0,0 +1,100 @@
+package mesosphere.marathon
+package api.akkahttp.v2
+
+import akka.http.scaladsl.server.Route
+import com.typesafe.scalalogging.StrictLogging
+import mesosphere.marathon.api.akkahttp.Controller
+import mesosphere.marathon.core.election.ElectionService
+import mesosphere.marathon.plugin.auth.{ Authenticator, AuthorizedResource, Authorizer, ViewResource }
+import mesosphere.marathon.storage.repository.FrameworkIdRepository
+import mesosphere.util.state.MesosLeaderInfo
+import play.api.libs.json._
+
+import scala.async.Async._
+import scala.concurrent.ExecutionContext
+
+case class InfoController(
+    val mesosLeaderInfo: MesosLeaderInfo,
+    val frameworkIdRepository: FrameworkIdRepository,
+    val config: MarathonConf)(
+    implicit
+    val authenticator: Authenticator,
+    val authorizer: Authorizer,
+    val electionService: ElectionService,
+    val executionContext: ExecutionContext) extends Controller with StrictLogging {
+
+  import mesosphere.marathon.api.akkahttp.Directives._
+  import mesosphere.marathon.api.akkahttp.EntityMarshallers._
+
+  // Marathon configurations
+  val marathonConfigValues = Json.obj(
```

Comment:
Let's move that somewhere else.

Reply:
I'll model the response in RAML and then use these classes.